src/osm/osmTagToNameMapping-cs.ts

```diff
@@ -1,397 +1,614 @@
+    cycleway: "Cyklostezka",
+    footway: 'Chodník',
+    living_street: "Rezidenční zóna",
+    motorway_link: "Napojení na dálnici",
+    motorway: "Dálnice",
+    path: "Pěšina",
+    primary_link: "Napojení na cestu první třídy",
+    primary: "Cesta první třídy",
+    residential: "Ulice",
+    secondary: "Cesta druhé třídy",
+    secondaty_link: "Napojení na cestu druhé třídy",
     service: {
-      '*': 'Servisní, příjezdová cesta',
+      '*': "Servisní, příjezdová cesta",
       service: {
-        '*': 'Servisní cesta {}',
-        alley: 'Příjezdová cesta',
-        bus: 'Cesta vyhražená pro autobusy',
-        driveway: 'Příjezdová cesta',
-        'drive-through': 'Cesta pro nákup z auta',
-        emergency_access: 'Požární cesta',
-        parking_aisle: 'Cesta parkoviště',
-      },
+        '*': "Servisní cesta {}",
+        alley: "Příjezdová cesta",
+        bus: "Cesta vyhražená pro autobusy",
+        driveway: "Příjezdová cesta",
+        'drive-through': "Cesta pro nákup z auta",
+        emergency_access: "Požární cesta",
+        parking_aisle: "Cesta parkoviště"
+      }
     },
-    construction: 'Cesta ve výstavbě',
-    crossing: 'Přechod',
-    cycleway: 'Cyklostezka',
-    footway: 'Chodník',
-    living_street: 'Rezidenční zóna',
-    motorway_link: 'Napojení na dálnici',
-    motorway: 'Dálnice',
-    path: 'Pěšina',
-    primary_link: 'Napojení na cestu první třídy',
-    primary: 'Cesta první třídy',
-    residential: 'Ulice',
-    secondary: 'Cesta druhé třídy',
-    secondaty_link: 'Napojení na cestu druhé třídy',
     steps: 'Schody',
-    tertiary_link: 'Napojení na cestu třetí třídy',
-    tertiary: 'Cesta třetí třídy',
-    trunk_link: 'Napojení na cestu pro motorová vozidla',
-    trunk: 'Silnice pro motorová vozidla',
-    unclassified: 'Neklasifikovaná cesta',
+    tertiary_link: "Napojení na cestu třetí třídy",
+    tertiary: "Cesta třetí třídy",
+    track: {
+      '*': "Lesní / polní cesta",
+      tracktype: {
+        grade1: "Zpevněná lesní / polní cesta",
+        grade2: "Převážně zpevněná lesní / polní cesta",
+        grade3: "Méňe pevná lesní / polní cesta",
+        grade4: "Převážně měkká lesní / polní cesta",
+        grade5: "Měkká lesní / polní cesta"
+      }
+    },
+    trunk_link: "Napojení na cestu pro motorová vozidla",
+    trunk: "Silnice pro motorová vozidla",
+    unclassified: 'Neklasifikovaná cesta'
   },
   boundary: {
-    '*': 'Oblast',
+    '*': "Oblast",
     administrative: {
-      '*': 'Administrativní oblast',
+      '*': "Administrativní oblast",
       admin_level: {
-        '10': 'Katastrální území',
+        '10': "Katastrální území",
         '9': 'Obec',
         '8': 'Okres',
-        '7': 'Oblast',
-        '6': 'Město',
-        '5': 'Provincie',
+        '7': "Oblast",
+        '6': "Město",
+        '5': "Provincie",
         '4': 'Kraj',
-        '3': 'Region',
-        '2': 'Stát',
-      },
+        '3': "Region",
+        '2': "Stát"
```

Comment:
prosim vrat naspat ciarky aj na poslednych zaznamoch. syntax musi prejst `prettier`-om (`npx prettier -w osmTagToNameMapping-cs.ts` - to opravi aj tie uvodzovky)